src/type/js/honeycomb.type.font.js

```diff
@@ -29,5 +31,5 @@ Honeycomb.Font = (function($) {
 })(jQuery);
```

Comment:
in line 21 we make a request for the font -- this won't work on DLM Dashboard and other apps on firewalled machines, where the browser can't talk to the internet but can talk to the on-premise install, which is why we ship Lato in our installer.  Also, dependency inversion principle suggests dependencies are injected in, rather than URLs for googleapis created here and used here...